tools/verifyio/verifyio.py

```diff
@@ -285,22 +377,25 @@ def build_call_chain_str(call_chain, reader):
     #print('6. RAM Used (GB):', psutil.virtual_memory()[3]/1000000000)
     print("Step 2. match mpi calls: %.3f secs, mpi edges: %d" %((t2-t1),len(mpi_edges)))
 
-    t1 = time.time()
-    vio.G = VerifyIOGraph(all_nodes, mpi_edges, include_vc=True)
-    t2 = time.time()
-    print("Step 3. build happens-before graph: %.3f secs, nodes: %d" %((t2-t1), vio.G.num_nodes()))
-    #print('7. RAM Used (GB):', psutil.virtual_memory()[3]/1000000000)
+    if vio.without_graph:
```

Comment:
we don't need this extra argument. you could simply check if vio.algorithm == 4.
only that algorithm does not need the graph.